### ISSUE TITLE
How to get going with a map integration?

### ISSUE THREAD

**Issue by krzkrzkrz** [tidwall/tile38#423]
How do we go about integrating this with a map? The documentation is pretty straightforward with adding/retrieving data into the tile38 server. 

But not very clear on how we can visualize this on a map

**Comment by amitkumar3968** [comment]
I think, this might help you.
https://medium.com/@s32x/mapping-11m-geonames-points-with-tile38-c9d326461b23

**Comment by krzkrzkrz** [comment]
Not quite. Came across that earlier too. Article speak from a general perspective. May have missed it, but I dont see anything anywhere about pulling out data from tile38 and loading it somewhere into a map

Maybe the question Im really asking is, how do I pull out all data stored in tile38 so I can display it in a map

**Comment by tidwall** [comment]
I'm assuming that you want to display objects on a Web Mercator map like Mapbox or Google Maps. I recommend querying using the TILE option.

```
INTERSECTS myobjs TILE X Y Z
```

This will return all the object overlapping the tile.  The XYZ coordinates are compatible with https://developers.google.com/maps/documentation/javascript/coordinates

You'll probably need some type middleware that sits between the browser and Tile38, which would translate the Tile38 results into the format needed from the JS API.

**Comment by sweco-semtne** [comment]
Hi! 
First of all I would like to thank you for the great work with Tile38! Thanks!
Would it be possible to extend the tile command to provide a tile size and have all geometries in the tile translated to a local tile reference?

So if you provide a XYZ TILESIZE, where TILESIZE would be an integer value, ie 4096, Tile38 would give you translated coordinates to relative the tile size. 
I guess the translation from XYZ to lat/lon bounds are already part of the code so this would be an extension to translate the coordinates on the way out. 

Bad idea?
